scripts/postbuild.js

```diff
@@ -1,3 +1,10 @@
 const fs = require('fs');
 
-fs.copyFileSync('./src/npm/index.js', './dist/index.js');
+function logErrors(err) {
+  if (err) {
+    throw err;
+  }
+}
+
+fs.copyFile('./src/npm/index.js', './dist/index.js', logErrors);
+fs.copyFile('./typescript/index.d.ts', './dist/index.d.ts', logErrors);
```

Comment:
I think this should be 
```ts
fs.copyFile('./typescript/index.d.ts', './dist/typescript/index.d.ts', logErrors);
```
Since in `package.json` we are mentioning `files` as `dist` and `types` as `typescript` directory. Hence we would need to have same directory structure inside `dist`.

Reply:
hey! Good catch-- I meant to remove the `types` property as we are just putting `index.d.ts` at the top level. Thanks!